app/dcs/third-party-tools/mountainduck/page.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Mountain Duck
+docId: Eem9ais3o-fee9iaquieh
+tags:
+  - file-management
+metadata:
+  title: Setting Up and Using Mountain Duck
+  description:
+    A guide on how to download, setup, and use the Mountain Duck
+    drive mounting tool to integrate and transfer files easily with Storj.
+---
+
+## Introduction
+
+Mountain Duck is an inexpensive, user-licensed application developed by the same team as [Cyberduck](docid:QQGzEDU8o-IodQWmr7xP3), serving as a versatile file access tool. 
+It allows users to mount server and cloud storage accounts as local disks in Finder on macOS and the File Explorer on Windows. 
+Supporting various protocols like FTP, SFTP, WebDAV, and cloud storage providers, including Storj, Mountain Duck provides a seamless integration of remote storage into the user's file system. 
+
+In this brief tutorial, we'll go over downloading and setting up Mountain Duck to integrate with Storj, facilitating easy and intuitive drag-and-drop file transfer to Storj.
+
+
+
+## Prerequisites:
+
+- Storj account with [](docId:AsyYcUJFbO1JI8-Tu8tW3)
+
+- Storj S3 compatiable access and secret key (see [](docId:AsyYcUJFbO1JI8-Tu8tW3))
+
+
+
+## Downloading Mountain Duck
+
+Users can download Mountain Duck by navigating to [](https://mountainduck.io/). Mountain Duck offers a free trial, so that you can evaluate the software before deciding if you'll purchase a license.
```

Comment:
Change `[](https://mountainduck.io/)` to `<https://mountainduck.io/>`. 

Only internal links with a `docId`, will populate a blank `[]`, as written this will render as nothing